docs/PIPELINES_CONFIGURATION.md

```diff
@@ -4,7 +4,7 @@ The OpenShift Pipelines configuration is a requirement in order to support CI/CD
 
 1. Install the [Openshift Pipelines Operator](https://docs.redhat.com/en/documentation/openshift_container_platform/4.6/html/pipelines/installing-pipelines#installing-pipelines).
 
-2. Ensure that the `pipeline-as-code-controller` is up by getting its route:
+2. Ensure that the `pipeline-as-code-controller` is up by getting its route. Remember to add this route as the webhook url of your Github app:
```

Comment:
nit
```suggestion
2. Ensure that the `pipeline-as-code-controller` is up by getting its route. Remember to add this route as the webhook url of your GitHub App:
```

Reply:
Addressed in [f19e02d](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/20/commits/f19e02d27583f22f67de4938fa5205423b8a7e3f)